skbio/metadata/io.py

```diff
@@ -0,0 +1,473 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2023, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
```

Comment:
Same here 
```
# ----------------------------------------------------------------------------
# Copyright (c) 2013--, scikit-bio development team.
#
# Distributed under the terms of the Modified BSD License.
#
# The full license is in the file LICENSE.txt, distributed with this software.
# ----------------------------------------------------------------------------
```

Reply:
We may want to have a discussion on where this file belong.  We may want to add this to the io registry.  See some of the examples in this directory : https://github.com/scikit-bio/scikit-bio/tree/master/skbio/io/format